app/components/project-long-description.js

```diff
@@ -1,13 +1,49 @@
 import Ember from 'ember';
 
+/**
+  `project-long-description` displays and allows editing of the long
+  long description for the project.
+
+  ## Default usage
+  ```handlebars
+    {{project-long-description project=project}}
+  ```
+
+  @module Components
```

Comment:
so far *to my knowledge* we have been using the singular version for `@module Component`.  Unfortunately, yuidoc is not *smart* enough to realize `Component` & `Components` are the same so they would not appear under the same module in the docs.